### ISSUE TITLE
Quill 2 Compatible?

### ISSUE THREAD

**Issue by KeithGillette** [artknight/quill-emoji#1]
Apologies for opening an issue for these questions, @artknight,  but I couldn't find another means of reaching out to ask. Is this a clone of `contentco/quill-emoji` updated for compatibility with Quill 2, now in release candidate status? We use `contentco/quill-emoji`, but it's not actively maintained, so I'm not expecting a Quill 2 compatible release anytime soon. As a result, I'm very interested in alternatives that would allow us to upgrade to Quill 2. Are you planning to publish this package on NPMJS or submit a pull request back to `contentco/quill-emoji`?

**Comment by artknight** [comment]
@KeithGillette yes, this is a clone of `contentco/..` repo. I had reached out to them on the issues with quill v2, and later on with updating their repo with the suggested fixes.. both times without any response. I am using quill v2, and all these plugins in my [Cogency](https://cogency.io) project, and will be maintaining the repos to continue being compatible with quill v2 going forward. Since my interest thus far has been purely superficial ( just making sure they are compatible w/out adding any new features ) I am not sure if publishing any of this is worthwhile. However, if you have ideas on how to keep the code best accessible to others please feel free to reach out.

**Comment by KeithGillette** [comment]
Thanks for the reply, Art. Sorry to hear but not surprised that you didn’t get any response from ContentCo, as I haven’t seen activity on the `quill-emoji` repository in a long time. At one time, the maintainers gave me repository merge access because I raised a small bugfix PR but I have no idea how to publish a new version.

Even if you’re just planning on maintaining the current functionality of this plugin for Quill 2 compatibility and security patches, I think it would be worth it to publish it on NPMJS so that others can discover it and track its releases. Given that others are still posting on the ContentCo repository, you may get community contributions to advance the package, as well.

**Comment by MentalGear** [comment]
Just wondering of the current status? If @KeithGillette hasn't updated the official quill-emoji, @artknight updated quill v2 emoji version on npm would certainly be much appreciated by the community !

**Comment by artknight** [comment]
@MentalGear I have been made a contributor to the original quill-emoji repo, and asked to port all my changes over. I haven't gotten a chance to do that yet, but it seems that the demand for it is building. I am going to do that this week.

**Comment by MentalGear** [comment]
Feel free to share the link to where you made your change offer so we can add our voices as well for added support.

**Comment by artknight** [comment]
@MentalGear Here you go, https://github.com/quill-mention/quill-mention/pull/369#issuecomment-2085635822

**Comment by KeithGillette** [comment]
That link was to `quill-mention` not `quill-emoji`, @artknight. However, it looks like you did raise a PR to port these changes over to `quill-emoji` on [PR#376](https://github.com/quill-mention/quill-mention/pull/376), but it seems there were some issues in review and I can't understand whether your changes for Quill 2 compatibility actually got integrated or released.
